### DISCUSSION TITLE
Release Notes CovPass(Check) Version 1.31

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-android#231] [Announcements]
# Release Notes CovPass(Check) Version 1.31

Release 1.31 is currently in final review and is expected to be released within the next couple of hours. 
  
### **Main Features CovPass:**  💉 

- Deactivating the update notifications: With the upcoming release, we will make it possible to view the update notifications at any time in the settings. Users who do not have time to study the updates in the course of a check situation can now view information about new functionalities via the settings. If you prefer to view updates via the settings, you can disable the update notification on first opening after the update. The text will still be readable in the settings. Excluded from the deactivation are so-called onboarding screens that introduce important new topics in detail to the users. These will continue to be played out as needed.

### **Bug Fixes CovPass:**  💉 

- (IOS only) Re-Priotization for revoked/ expired DCCs: If multiple blocked or expired DCCs were available for an individual, they were prioritized by scan date rather than by vaccination date/date of the underlying test. This deviating prioritization has been fixed.

-----------------------------------------------------------------------------------------------------------

### **Main Features CovPassCheck:**  💉 

- Saved check settings: In CovPassCheck your check settings are now permanently stored. When the app is opened again, the checkers will find familiar check settings.

- Acoustic Feedback: Verifiers can now activate acoustic feedback via the CovPassCheck setting. When certificates are scanned, a signal tone sounds that is independent of the check result. This is intended to reassure the auditee that the scan was performed using the CovPassCheck provided for the checking purpose.

- Deactivation of Offline-Revocation-Checks must be confirmed:  Verifiers who want to deactivate the offline check for blacklist entries must now confirm the deactivation manually, as deactivation leads to the deletion of the saved lists. Reactivation is accompanied by a new download of the revocation lists.

### **Bug Fixes CovPassCheck:**  💉 

- Alignment of platforms: Minor visual differences between the IOS and Android implementations have been fixed.

-----------------------------------------------------------------------------------------------------------

An outlook for upcoming topics will be published soon!

-----------------------------------------------------------------------------------------------------------

**Thank you all for you support and feedback! Stay safe everyone** 👍 🎊 

Team CovPass and CovPassCheck! 🧑‍⚕️ 👩‍⚕️

**Comment by MikeMcC399** [comment]
- https://play.google.com/store/apps/details?id=de.rki.covpass.app and
- https://play.google.com/store/apps/details?id=de.rki.covpass.checkapp

are still showing 1.30.0 as the latest version of CovPass and CovPass Check

**Reply by MikeMcC399** [reply]
Google Play Store is now serving the 1.31.0 version of both apps.

**Reply by Ein-Tim** [reply]
@MikeMcC399 do you mean 1.31?

**Reply by MikeMcC399** [reply]
@Ein-Tim Thanks! Typo corrected!

**Comment by Ein-Tim** [comment]
@alexcimander 

Will you release the source code branch soon?
